### PR TITLE
Issue work order

### PR DESCRIPTION
**Pull request by pudiva**
Trello:
https://trello.com/c/q1S6dgIg/131-as-an-rcc-agent-after-raising-a-work-order-in-repairs-hub-i-need-to-find-that-work-order-in-drs-so-that-i-can-book-an-appointmen

### CODE REVIEW

File: app/controllers/repair_requests_controller.rb [hunk 1]
```diff
@@ -44,6 +45,33 @@ def create
 
   private
 
+  def issue_dlo_work_orders
+    n_errors = 0
```
Comment (cdccollins) [inline app/controllers/repair_requests_controller.rb:49]: is this number_of_errors?

Comment (pudiva) [reply app/controllers/repair_requests_controller.rb:49]: What do you mean? The variable is `n_errors` and it counts the number of API errors that happened trying to issue the work orders.

Comment (cdccollins) [reply app/controllers/repair_requests_controller.rb:49]: I just mean the name was a little bit confusing when I first came across it

Comment (pudiva) [reply app/controllers/repair_requests_controller.rb:49]: do you want me to rename it?

Comment (cdccollins) [reply app/controllers/repair_requests_controller.rb:49]: meh, it's probably ok

Comment (pudiva) [reply app/controllers/repair_requests_controller.rb:49]: then will you approve it? :upside_down_face:

**Review by cdccollins — APPROVED** [review]

**Pull request merged**